app/src/main/java/com/github/se/polyfit/ui/screen/NutritionScreen.kt

```diff
@@ -46,7 +47,8 @@ fun NutritionScreen(
         BottomBar(
             setMeal = mealViewModel::setMeal,
             isComplete = isComplete,
-            navigateForward = navigateForward)
+            navigateForward = navigateForward,
+            mealViewModel = mealViewModel)
```

Comment:
I think it is generally better to pass the method only (see `setMeal` above) instead of the whole mealViewModel
```suggestion
            updateMealData = mealViewModel::updateMealData)
```

Reply:
can you do something like `updateMealData = { createdAt ->  mealViewmodel.updateMealData(createdAt = createdAt) }`